Code/GraphMol/MolDraw2D/DrawTextFTSVG.cpp

```diff
@@ -8,10 +8,16 @@
 // Original author: David Cosgrove (CozChemIx) on 08/05/2020.
 //
 
+#include <boost/format.hpp>
 #include <GraphMol/MolDraw2D/DrawTextFTSVG.h>
 
 namespace RDKit {
 
+namespace {
+std::string formatDouble(double val) {
+  return str(boost::format("%.1f") % val);
```

Comment:
I think it'd be better if this were a bit more explicit:
```suggestion
  return boost::str(boost::format("%.1f") % val);
```